app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -48,6 +48,39 @@ function buildArticleHTML(article, currentUserId = null) {
       </article>`;
   }
 
+  if (article && article.class_name === 'Organization') {
+    return `<article class="crayons-story">
+        <div class="crayons-story__body flex items-start gap-2">
+          <a href="${article.slug}" class="crayons-podcast-episode__cover">
+            <img src="${article.nav_image.url}" alt="${
+      article.name
+    }" loading="lazy" />
+          </a>
+          <div>
+            <h3 class="crayons-subtitle-2 lh-tight py-1">
+              <a href="${article.slug}" class="c-link">
+                ${article.name}
+              </a>
+            </h3>
+            <p>@${article.twitter_username}</p>
```

Comment:
This should be their on-site username (i.e. slug for organizations), rather than Twitter username.